### PR TITLE
fix(api-extensions): remove references to BONITA_HOME

### PR DESCRIPTION
**Pull request by redboul**
Closes [BS-15735](https://bonitasoft.atlassian.net/browse/BS-15735)

### CODE REVIEW

File: md/api-extensions.md [hunk 1]
```diff
@@ -52,8 +52,10 @@ A REST API extension example resource and API extension viewer page are provided
 
 To use the examples, define a [custom permissions](rest-api-authorization.md#custom-permissions-mapping) mapping for `demoPermission`:
 
-* Edit `$BONITA_HOME/client/tenants/1/conf/custom-permissions-mapping.properties`.
+* Retrieve platform configuration using the [setup tool](BonitaBPM_platform_setup.md#update-platform-conf).
```
Comment (educhastenier) [inline md/api-extensions.md]: hash is incorrect: _update_platform_conf_

File: md/api-extensions.md [hunk 2]
```diff
@@ -52,8 +52,10 @@ A REST API extension example resource and API extension viewer page are provided
...
+* Edit `$PLATFORM_CONF/current/tenants/1/tenant_portal/custom-permissions-mapping.properties`.
```
Comment (educhastenier) [inline md/api-extensions.md]: Did we explain somewhere **$PLATFORM_CONF** ?

Comment (educhastenier) [inline md/api-extensions.md]: Maybe replace tenant 1 by `<your_tenant_id>`

Comment (redboul) [reply md/api-extensions.md]: it is used elsewhere but I do not know if we have

Comment (educhastenier) [reply md/api-extensions.md]: Even if used elsewhere, I think we should never need to navigate to another page to understand the meaning of a variable. A doc page should be self-sufficient in most cases.

Comment (educhastenier) [reply md/api-extensions.md]: Ok, after comparing with other pages `platform_conf/current/tenants/1/...` is enough.

**Comment by educhastenier** [comment]
Commentaire integration slack - github

File: md/api-extensions.md (hunk 2 above)
Comment (redboul) [reply md/api-extensions.md]: so you think we need to explain this $PLATFORM_CONF here?

Comment (educhastenier) [reply md/api-extensions.md]: no, just change `$PLATFORM_CONF` by `platform_conf` and that's enough

**Comment by tnguyen1** [comment]
retest this please

**Pull request merged**